server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -252,6 +252,12 @@ def _merge_first_steps(
 
         return ctx
 
+    def _ensure_term_uses_wrapper(self: Self, term: Term):
```

Comment:
This is the real fix ! This is ugly but unfortunately there is no easy way to fix this
(see https://github.com/kayak/pypika/issues/622)